### PR TITLE
Adding graceful error page for Kiosks

### PR DESCRIPTION
**Pull request by ouchadam**
| Before | After |
| --- | --- |
screen shot 2018-04-11 at 08 47 23|screen shot 2018-04-10 at 23 23 29

### CODE REVIEW

File: site/src/kiosk/error-view.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            <p>
+                Plugin failed to provide content.
+            </p>
+            <p style={content}>{error.cause}</p>
+            <p style={content}>
+                {error.source.split('?')[0]}
+            </p>
+            <p>
+            </p>
+            <p>#dashboards / #g-it</p>
```
Comment (takecare) [inline site/src/kiosk/error-view.jsx]: i'd remove this

File: site/src/kiosk/kiosk-use-case.js [hunk 2]
```diff
@@ -15,15 +16,15 @@ export const watchAnonymousStateChange = (dispatch, auth) => () => {
...
 export const watchDeviceContent = (dispatch) => (deviceId) => {
     return watchDeviceData(deviceId, (url) => {
-        fetch(url)
-            .then(response => response.text())
-            .then(html => {
-                dispatch(Actions.onDeviceContent(html))
-            })
+        loadHtml(url).then(html => {
+            dispatch(Actions.onDeviceContent(html))
+        }).catch((error) => {
```
Comment (takecare) [inline site/src/kiosk/kiosk-use-case.js]: just a nitpick, but i've seen some declarations where you wrap the single param with `()` and others where you don't. maybe we could setup a linter? 😄

**Review by takecare — APPROVED** [review]

File: site/src/kiosk/kiosk-use-case.js (hunk 2 above)
Comment (ouchadam) [reply site/src/kiosk/kiosk-use-case.js]: I'm terrible for this, yeah I agree. Do you have a preference for which one to choose? Always wrap or only for multiple params?

File: site/src/kiosk/error-view.jsx (hunk 1 above)
Comment (ouchadam) [reply site/src/kiosk/error-view.jsx]: How's about `Report to your dashboard administrator if the problem persists` instead?

Comment (takecare) [reply site/src/kiosk/error-view.jsx]: sounds good 👍

File: site/src/kiosk/kiosk-use-case.js (hunk 2 above)
Comment (takecare) [reply site/src/kiosk/kiosk-use-case.js]: my personal preference is only for multiple params. but don't mind if you go the other way.

**Pull request merged**